proto/mls/message_contents/message.proto

```diff
@@ -0,0 +1,32 @@
+// V3 invite message structure
+syntax = "proto3";
+
+package xmtp.mls.message_contents;
+
+option go_package = "github.com/xmtp/proto/v3/go/mls/message_contents";
+option java_package = "org.xmtp.proto.mls.message.contents";
+
+// Welcome messages are sent to an installation's welcome topic and are
+// encrypted using Noise Pattern N
```

Comment:
What metadata are we trying to hide here? Curious why we need to add this on top of MLS

Reply:
Maybe I was mistaken. I thought there was some sensitive metadata unencrypted in the Welcome. But looking a the code, it's [all encrypted](https://github.com/openmls/openmls/blob/2d4991b53feb3f8490ca23fbb5a5ccfff094d07d/openmls/src/messages/mod.rs#L66). The only thing they really expose is the `KeyPackageRef` of the recipient, and the ciphersuites used. That's not sensitive at all.

Reply:
Will remove references to Noise for now and leave the Welcomes unencrypted. We may have to re-add if we add our own metadata around the Welcome for some reason (not sure if we have to validate the sender with a signature to know who invited you)